bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```diff
@@ -468,6 +469,22 @@ channel-type.shelly.sensorWakeup.state.option.poweron = Power-On
 channel-type.shelly.sensorWakeup.state.option.sensor = Sensor
 channel-type.shelly.sensorWakeup.state.option.alarm = Alarm
 channel-type.shelly.sensorWakeup.state.option.ext_power = Charger connected
+channel-type.shelly.sensorRain.label = Rain Status
+channel-type.shelly.sensorRain.description = ON: It's raining, OFF: It's not raining
+channel-type.shelly.sensorWindSpeed.label = Wind Speed
+channel-type.shelly.sensorWindSpeed Mesaure Wind Speed in m/s
```

Comment:
`=` is missing and when a description only the first character should be uppercase